pkg/images/utils/utils.go

```diff
@@ -264,3 +264,26 @@ func FilterSuppressedCVEsNoClone(img *storage.Image) {
 		}
 	}
 }
+
+func IsRedHatImage(img *storage.Image) bool {
+	// First consider registries where all images are built by Red Hat
+	imageRegistry := img.GetName().GetRegistry()
+	redHatRegistries := set.NewStringSet(
```

Comment:
nit: can make this a global var, so it's only initialized once same for `"quay.io"` and the "remotes" below